osgeo_importer/views.py

```diff
@@ -1,20 +1,27 @@
 import os
 import json
 import logging
+import shutil
 from django.http import HttpResponse
 from django.views.generic import FormView, ListView, TemplateView
 from django.core.urlresolvers import reverse_lazy
 from .forms import UploadFileForm
-from .models import UploadedData, UploadLayer, DEFAULT_LAYER_CONFIGURATION
+from .models import UploadedData, UploadLayer, UploadFile, DEFAULT_LAYER_CONFIGURATION
 from .importers import OSGEO_IMPORTER
 from .inspectors import OSGEO_INSPECTOR
 from .utils import import_string
 from django.conf import settings
+from django.core.files.storage import FileSystemStorage
 
 OSGEO_INSPECTOR = import_string(OSGEO_INSPECTOR)
 OSGEO_IMPORTER = import_string(OSGEO_IMPORTER)
 
 logger = logging.getLogger(__name__)
+if logger.handlers:
+    for handler in logger.handlers:
+        logger.removeHandler(handler)
+logging.basicConfig(filename='/vagrant/views.log', level=logging.DEBUG)
```

Comment:
I gather that this is actually removed in later PR commits, so it's trivial - just something unsquashed